InventoryApp/Services/QueueClient.cs

```diff
@@ -1,45 +1,45 @@
-﻿using Microsoft.WindowsAzure.Storage.Queue;
+﻿using Azure.Storage.Queues;
 using Newtonsoft.Json;
-using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 using TrafficCaseApp.Models;
 
 namespace TrafficCaseApp.Services
 {
-    public class QueueClient : IQueueClient
+    public class QueueClientTem : IQueueClient
```

Comment:
Why was the class name changed? What does "Tem" mean?

Reply:
Because there's another QueueClient in Azure.Storage.Queues, Tem means temp.